elixir/apps/web/lib/web/live/policies/index.ex

```diff
@@ -88,4 +94,13 @@ defmodule Web.Policies.Index do
     </div>
     """
   end
+
+  def handle_event("delete", %{"id" => id}, socket) do
+    with {:ok, policy} <- Policies.fetch_policy_by_id(id, socket.assigns.subject) do
+      {:ok, _} = Policies.delete_policy(policy, socket.assigns.subject)
+      {:noreply, redirect(socket, to: ~p"/#{socket.assigns.account}/policies")}
```

Comment:
Out of curiosity, why [`redirect/2`](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#redirect/2) and not [`push_navigate/2`](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#push_navigate/2) here? I believe the former causes a full page reload.

Reply:
Ah, good question/catch.  I think this was again Copilot and since it worked I didn't really think critically about it.  I believe your correct about the full page reload, so I'll update this.

Reply:
Updated this to use `push_navigate`